Octokit/Clients/IRepositoryCommentsClient.cs

```diff
@@ -19,74 +19,141 @@ public interface IRepositoryCommentsClient
         /// <param name="owner">The owner of the repository</param>
         /// <param name="name">The name of the repository</param>
         /// <param name="number">The comment id</param>
-        /// <returns></returns>
+        /// <returns>A <see cref="CommitComment"/> for the specified comment id.</returns>
         [SuppressMessage("Microsoft.Naming", "CA1716:IdentifiersShouldNotMatchKeywords", MessageId = "Get",
             Justification = "Method makes a network request")]
         Task<CommitComment> Get(string owner, string name, int number);
 
         /// <summary>
+        /// Gets a single Repository Comment by number.
+        /// </summary>
+        /// <remarks>http://developer.github.com/v3/repos/comments/#get-a-single-commit-comment</remarks>
+        /// <param name="repositoryId">The ID of the repository</param>
+        /// <param name="number">The comment id</param>
+        /// <returns>A <see cref="CommitComment"/> for the specified comment id.</returns>
+        [SuppressMessage("Microsoft.Naming", "CA1716:IdentifiersShouldNotMatchKeywords", MessageId = "Get",
+            Justification = "Method makes a network request")]
+        Task<CommitComment> Get(int repositoryId, int number);
+
+        /// <summary>
         /// Gets Commit Comments for a repository.
         /// </summary>
         /// <remarks>http://developer.github.com/v3/repos/comments/#list-commit-comments-for-a-repository</remarks>
         /// <param name="owner">The owner of the repository</param>
         /// <param name="name">The name of the repository</param>
-        /// <returns></returns>
+        /// <returns>A <see cref="IReadOnlyPagedCollection{CommitComment}"/> of <see cref="CommitComment"/>s for the specified repository.</returns>
```

Comment:
This method is technically returning `Task<IReadOnlyList<CommitComment>>` but I'm not really familiar enough with the markup in here to confirm what this should look like 😕 

Something like this?

> A <see cref="Task{IReadOnlyPagedCollection{CommitComment}}"/> for the specified repository. 

There's a few places where this is setup. Let me know if this doesn't work and I can dig deeper.